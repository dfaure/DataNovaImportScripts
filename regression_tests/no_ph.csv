#Identifiant;Libellé_du_site;Date_calendrier;Plage_horaire_1;Plage_horaire_2;Plage_horaire_3;Plage_horaire_4;Plage_horaire_5;Plage_horaire_6;Plage_horaire_7;Heure_limite_dépôt_Courrier;Heure_limite_dépôt_Chrono;Heure_limite_dépôt_Colis;Libellé_de_la_fermeture;Bp_Plage_horaire_1;Bp_Plage_horaire_2;Bp_Plage_horaire_3;Bp_Plage_horaire_4;Bp_Plage_horaire_5;Bp_Plage_horaire_6;Bp_Plage_horaire_7;Bp_Libellé_de_la_fermeture
12504D;RPU AUCHAN SARROLA RP;2021-06-24;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-10;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-07;08:00-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-25;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-17;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-23;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-26;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-28;08:00-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-13;08:30-13:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-22;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-21;08:00-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-08;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-19;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-05;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-20;08:30-13:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-02;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-03;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-01;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-15;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-30;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-14;08:00-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-29;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-02;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-19;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-05-25;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-05-31;08:00-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-05-27;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-01;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-21;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-27;08:30-13:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-05-30;08:30-13:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-16;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-05-26;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-12;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-05;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-05-29;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-05-24;08:00-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-14;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-08;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-06;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-07;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-09;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-10;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-04;08:30-13:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-11;08:30-13:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-14;08:00-19:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-16;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-05;08:00-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-20;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-13;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-03;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-30;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-15;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-27;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-28;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-22;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-04;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-25;08:30-13:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-12;08:00-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-11;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-04;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-02;08:00-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-17;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-18;08:30-13:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-29;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-01;08:30-13:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-19;08:00-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-10;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-20;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-24;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-23;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-18;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-11;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-13;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-12;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-26;08:00-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-07-31;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-15;08:00-13:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-24;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-09;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-21;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-22;08:30-13:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-06-06;08:30-13:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-05-28;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-03;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-17;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-25;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-07;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-09;08:00-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-23;08:00-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-16;08:00-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-18;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-06;08:30-21:00;;;;;;;;;;;FERME;;;;;;;
12504D;RPU AUCHAN SARROLA RP;2021-08-08;08:30-13:00;;;;;;;;;;;FERME;;;;;;;
