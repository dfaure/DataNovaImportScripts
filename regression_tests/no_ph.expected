12504D|RPU AUCHAN SARROLA RP|Mo 08:00-21:00; Tu-Sa 08:30-21:00; Su 08:30-13:00; PH off; 2021 Aug 15 08:00-13:00; 2021 Jul 14 08:00-19:00; 2021 May 24 08:00-21:00
